ledger/src/bank_forks_utils.rs

```diff
@@ -98,72 +99,95 @@ pub fn load_bank_forks(
     LeaderScheduleCache,
     Option<StartingSnapshotHashes>,
 ) {
-    let snapshot_present = if let Some(snapshot_config) = snapshot_config {
-        info!(
-            "Initializing bank snapshot path: {}",
-            snapshot_config.bank_snapshots_dir.display()
-        );
-        fs::create_dir_all(&snapshot_config.bank_snapshots_dir)
-            .expect("Couldn't create snapshot directory");
+    fn get_snapshots_to_load(
+        snapshot_config: Option<&SnapshotConfig>,
+    ) -> Option<(
+        FullSnapshotArchiveInfo,
+        Option<IncrementalSnapshotArchiveInfo>,
+    )> {
+        let Some(snapshot_config) = snapshot_config
```

Comment:
nit: https://doc.rust-lang.org/1.69.0/style-guide/statements.html#single-line-let-else-statements

It seems the rust style-guide indicates this should have the `else {` on the same line as `snapshot_config`.
wish cargo-fmt implemented these rules!